JDBC/DeleteBookPOJO.java

```diff
@@ -0,0 +1,11 @@
+package com.metacube.jdbc.example;
+
+public class DeleteBookPOJO {
```

Comment:
ACTION: Eventhough it is a pojo class we don't specify POJO in class name.Use some other class name.